blog/content/post/2020-05-14-ipxe_lz_support.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Starting TrenchBoot's Landing Zone from iPXE
+abstract: 'In this article we present support for starting Landing Zone from
+          another bootloader: iPXE. It may not be as featureful as GRUB2, but
+          it has enough juice to start DRTM using images obtained from a remote
+          server'
+cover: /covers/trenchboot-logo.png
+author: krystian.hebel
+layout: post
+published: true
+date: 2020-05-14
```

Comment:
Do we want to change the date?

Reply:
Changed to today's date

Reply:
I always update the date and filename-date before the publishing